v1/src/test/java/com/google/cloud/teleport/spanner/ddl/DdlTest.java

```diff
@@ -864,6 +865,25 @@ public void pgSequences() {
     assertNotNull(ddl.hashCode());
   }
 
+  @Test
+  public void protobundle() {
```

Comment:
Lets say if the customer already has create a proto bundle with types A and B.
The data in import files contain proto types C and D. The import will try to run CREATE PROTO BUNDLE statement. Does this statement append to the existing bundle? Or does it overwrite? Or does the statement fail to execute?

Reply:
Spanner allows us to provide either individual ddl statements or complete ddl file. Dataflow templates provide complete ddl file through updateddl so it'll be a modified CREATE PROTO BUNDLE statement with additional proto types added to it. Spanner internally compares and makes a call whether to use alteration or creation. This is same for all ddl objects.